src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -733,6 +749,28 @@ public function collectFromCode($content, $fileName, Module $module)
                     continue;
                 }
 
+                // Allow _t(Entity.Key, 'translation', $var) and expand _t(Entity.Key, $var)
+                if ($id == T_VARIABLE && !empty($currentEntity)) {
+                    // We have a translation, eg: _t(Entity.Key, 'translation', $var)
+                    if (count($currentEntity) == 2) {
+                        continue;
+                    }
+
+                    // The variable is the second argument or present in the parameter
+                    // Try to find it _t(Entity.Key, $var)
+                    $stringValue = $stringVariables[$text] ?? null;
+
+                    // It's translated, continue
```

Comment:
By "It's translated" do you mean "We've fully captured this localisation key and its default text"? If so, please change the comment to something in that vain so it's clear what you mean.

Reply:
it's translated in this case means a translation has been provided and need to be processed. by default captured string look like 'my translation' and need to go through processString before being stored

Reply:
yes indeed, i've updated the comments